jirazzz

```diff
@@ -276,8 +276,8 @@
                    {:json {:fields (merge
                                      (select-keys opts [:summary
                                                         :description])
-                                     {:project {:id (:project-id opts)}
-                                      :issuetype {:id (:type-id opts)}
+                                     {:project {:key (:project-id opts)}
```

Comment:
https://docs.atlassian.com/software/jira/docs/api/REST/9.1.1/#api/2/issue-createIssue

This api doc still shows that `id` is the parameter for the request.
It is spelled `key` in the response.
(These are request parameters).